### DISCUSSION TITLE
Publish on Amazon Appstore

### DISCUSSION THREAD

**Discussion by shuvashish76** [TeamAmaze/AmazeFileManager#2879] [General]
Amazon Appstore: https://www.amazon.com/gp/mas/get/android/

---
Here are the top 3 results for how to publish app on Amazon Appstore

https://developer.amazon.com/docs/app-submission/submitting-apps-to-amazon-appstore.html

https://developer.amazon.com/apps-and-games/app-submission

https://developer.amazon.com/blogs/appstore/post/Tx3D619VJVL6NC1/submitting-apps-to-the-amazon-appstore-developer-portal

---

https://www.geeksforgeeks.org/how-to-publish-your-android-app-on-amazon-app-store-for-free/

**Comment by VishnuSanal** [comment]
I think someone has published it [here](https://www.amazon.in/Coded-Brains-Amaze-File-Manager/dp/B010BT4W9Y)! Not us, AFAIK.

**Reply by shuvashish76** [reply]
Developed By: **Coded Brains**
Developer info: **brains.coded@gmail.com**

Isn't it comes under License fraud #1529

[Amaze File Manager](https://www.amazon.com/dp/B010BT4W9Y/ref=apps_sf_sta) (com.amaze.filemanager) 1.3.1 (5)

@EmmanuelMess

**Reply by EmmanuelMess** [reply]
Yes

**Comment by EmmanuelMess** [comment]
We should have an official on Amazon store copy before adding this to "clones". We need to DMCA the current com.amaze.filemanager for that or use another package name.

**Comment by VishalNehra** [comment]
I've sent multiple mails to Amazon but they don't care

**Reply by EmmanuelMess** [reply]
Can we DMCA the current com.amazefilemanager without going through Amazon?

**Reply by shuvashish76** [reply]
Any updates @VishalNehra ?

**Reply by VishalNehra** [reply]
i'll give it a try again. I'm not sure how we can DMCA when even that platform itself isn't responding to any of the developer's queries.

**Comment by EmmanuelMess** [comment]
At this point we should just change the package name on Amazon.

**Reply by EmmanuelMess** [reply]
It'd work similar to debug version.
